data/spells/lib/spells.lua

```diff
@@ -309,42 +309,53 @@ function Player:addPartyCondition(combat, variant, condition, baseMana)
 		return false
 	end
 
+	local positions = {}
+	function onTargetTile(creature, position)
+		positions[#positions + 1] = position
+	end
+
+	combat:setCallback(CALLBACK_PARAM_TARGETTILE, "onTargetTile")
```

Comment:
I don't know enough to understand such particularities, but is this a good idea (redefining the tile callback function and calling ``setCallback`` each time such spells are cast.) The main reason I did this is to remove common duplicated code from the four spells (3a5a167).

@djarek